packages/communication-react/src/index.ts

```diff
@@ -141,6 +141,9 @@ export type {
   EndCallButtonStrings,
   ErrorBarProps,
   ErrorBarStrings,
+  FileCardStrings,
```

Comment:
File sharing components are internal components, should we still be exporting it's strings?

Reply:
Although file sharing components are internal, we export these strings for localization purposes.